package.json

```diff
@@ -55,16 +55,19 @@
         "configuration": {
             "title": "Live Sass Compile Config",
             "properties": {
-                "liveSassCompile.settings.format": {
-                    "type": "string",
-                    "default": "expanded",
-                    "enum": [
-                        "expanded",
-                        "compact",
-                        "compressed",
-                        "nested"
+                "liveSassCompile.settings.formats": {
+                    "type": "array",
+                    "default": [
+                        {
+                            "format": "expanded",
+                            "extensionName": ".css"
+                        },
+                        {
+                            "format": "compressed",
+                            "extensionName": ".min.css"
+                        }
                     ],
```

Comment:
I think, default value should be for one type. [NOT EVERYONE WANTS MULTIPLE FORMAT.]
I mean. by default extension will produce one type of `css`. If user need more, they have to just add more.

What do you think?

Reply:
Agreed. Just i left it to show how to use multiple. Feel free to change anything. I tried to do it how i understood the code, but is your architecture, so is better if you adapt the code to your like.